delivery_auto_refresh/models/res_company.py

```diff
@@ -7,12 +7,6 @@
 class ResCompany(models.Model):
     _inherit = "res.company"
 
-    # Migration Note 17.0: move this to module sale_order_carrier_auto_assign
-    sale_auto_assign_carrier_on_create = fields.Boolean(
```

Comment:
We need migration script for this, as now it has another name.

Reply:
Changes done.